modules/core/src/test/scala/org/tessellation/infrastructure/trust/storage/TrustStorageSuite.scala

```diff
@@ -0,0 +1,361 @@
+package org.tessellation.infrastructure.trust.storage
+
+import cats.syntax.option._
+import cats.syntax.partialOrder._
+import cats.syntax.semigroup._
+
+import org.tessellation.cli.method.{delayToUpdateCurrent, snapshotInterval}
+import org.tessellation.domain.trust.storage.{OrdinalTrustMap, PublicTrustMap, TrustMap}
+import org.tessellation.infrastructure.trust.generators.{genPeerLabel, genPeerObservationAdjustmentUpdate, genPeerTrustInfo}
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.schema.generators.peerIdGen
+import org.tessellation.schema.trust._
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.auto._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.numeric.NonNegative
+import org.scalacheck.Gen
+import weaver.SimpleIOSuite
+import weaver.scalacheck.Checkers
+
+object TrustStorageSuite extends SimpleIOSuite with Checkers {
+  test("initial trust storage is empty") {
+    for {
+      store <- TrustStorage.make
+      trust <- store.getTrust
+    } yield expect.all(TrustMap.empty == trust)
```

Comment:
equality check should be done using `Eq` typeclass (`===`)

Reply:
All equality in the tests have been updated to use `===`.